selene-lib/default_std/roblox_base.yml

```diff
@@ -107,6 +107,17 @@ globals:
     must_use: true
   CFrame.identity:
     property: read-only
+  CFrame.lookAlong:
+    args:
+      - type:
+          display: Vector3
+      - type:
+          display: Vector3
+      - required: false
+      - type:
```

Comment:
```suggestion
        type:
```

Reply:
I see... I thought testing was only available for lints but if it works for this as well, I'll get right to it!